src/cookieconsent.js

```diff
@@ -116,7 +116,7 @@
 
     // used to change color on highlight
     getLuminance: function(hex) {
-      var num = parseInt(this.normaliseHex(hex), 16), 
```

Comment:
Sorry for the automatic formatting changes from my editor, I can remove them if desired.